packages/react-server-test-pages/pages/navigation/playground.js

```diff
@@ -33,27 +33,58 @@ const PagePointer = ({page, row}) => <div className="page-pointer">
 	{+page === +row ? "➟" : ""}
 </div>;
 
-const NormalLink = ({row}) => <a href={LINK(row)}>Normal Link</a>
-
-const ClientTransitionLink = ({row}) => <Link path={LINK(row)}>Client Transition</Link>
-
-const FramebackLink = ({row}) => <Link frameback={true} path={LINK(row)}>Frameback</Link>
-
-const ReuseDom = ({row}) => <Link reuseDom={true} path={LINK(row)}>Reuse DOM</Link>
-
-const BundleData = ({row}) => <Link bundleData={true} path={LINK(row)}>Bundle Data</Link>
+const NL       = ({row}) => <a href={LINK(row)}>Normal Link</a>
+const CT       = ({row}) => <Link path={LINK(row)}>CT</Link>
+const RD       = ({row}) => <Link reuseDom={true} path={LINK(row)}>CT/RD</Link>
+const BD       = ({row}) => <Link bundleData={true} path={LINK(row)}>CT/BD</Link>
+const BDRD     = ({row}) => <Link bundleData={true} reuseDom={true} path={LINK(row)}>CT/BD/RD</Link>
+const FB       = (props) => <FBL {...props}></FBL>
+const FBCT     = (props) => <FBL {...props} link={{reuseFrame:true}}>CT</FBL>
+const FBCTBD   = (props) => <FBL {...props} link={{reuseFrame:true, bundleData:true}}>CT/BD</FBL>
+const FBCTRD   = (props) => <FBL {...props} link={{reuseFrame:true, reuseDom:true}}>CT/RD</FBL>
+const FBCTBDRD = (props) => <FBL {...props} link={{reuseFrame:true, bundleData:true, reuseDom:true}}>CT/BD/RD</FBL>
```

Comment:
Have a lot of links now.  Gave them shorter names.  Added a legend to the page.

![image](https://cloud.githubusercontent.com/assets/115908/14577463/3d16fd36-0329-11e6-948c-6d7c77d43696.png)